sdks/python/setup.py

```diff
@@ -101,6 +101,7 @@ def get_version():
 
 REQUIRED_TEST_PACKAGES = [
     'pyhamcrest>=1.9,<2.0',
+    'Sphinx>=1.5.2,<2.0'
```

Comment:
We can add it as an extra_require too, will update. I added it here that it was used only in dev.